cpg_qc/utils.py

```diff
@@ -29,6 +29,27 @@ def init_hail(name: str, local_tmp_dir: str):
     hl.init(default_reference=DEFAULT_REF, log=hl_log)
 
 
+def init_hail_service(
```

Comment:
Did you mean to include this in this PR or is it only testing code for Hail Query within Hail Batch?

Reply:
That's just to be ready to switch to the service backend as soon as it's working. But removed for now :)